### PR TITLE
Add vSphere disconnected profile

### PR DESCRIPTION
**Pull request by sgaoshang**
Add vSphere disconnected profile for IBMC

### CODE REVIEW

**Comment by sgaoshang** [comment]
@jinyunma @jianlinliu pls help to review it, thanks.

File: core-services/prow/02_config/generate-boskos.py [hunk 1]
```diff
@@ -295,6 +296,9 @@
 for i in range(60,64):
     CONFIG['vsphere-discon-quota-slice']['qe-discon-segment-{}'.format(i)] = 1
 
+for i in range(230,235):
+    CONFIG['vsphere-dis-quota-slice']['devqe-segment-{}'.format(i)] = 1
```
Comment (jianlinliu) [inline core-services/prow/02_config/generate-boskos.py:300]: shall we add `discon` keyword in `devqe-segment` ? Or `devqe-segment` is the name configured in ibmc vsphere cluster, can not be changed.

Comment (jinyunma) [reply core-services/prow/02_config/generate-boskos.py:300]: seems missing `-disconnected` here? should be `devqe-segment-{}-disconnected`.

Comment (sgaoshang) [reply core-services/prow/02_config/generate-boskos.py]: Good catch, I added it back, thanks.

**Pull request merged**